graphql/version.py

```diff
@@ -18,9 +18,9 @@ def __str__(self):
         return v
 
 
-version = "1.0.5"
+version = "3.0.0"
```

Comment:
There are som more places with the version number, see .bumpversion.cfg

Reply:
Ok, I did a find and replace to update the remaining versions.